orchestration/dags/data_gcp_dbt/models/intermediate/firebase/native/int_firebase__native_event_flattened.sql

```diff
@@ -146,5 +146,14 @@ select
         venues_41_50
     ),
     {{ extract_str_to_array_field("offers", 0, 10, 50) }} as displayed_offers,
-    {{ extract_str_to_array_field("venues", 0, 10, 50) }} as displayed_venues
+    {{ extract_str_to_array_field("venues", 0, 10, 50) }} as displayed_venues,
+    case
```

Comment:
(event_name = 'BookingConfirmation' and bookingid is null) or (when event_name = 'BookingConfirmation' and user_id is null) as is_anomaly

Reply:
pourquoi ? pas sure de comprendre
je trouve ça plus facile avec case when pour la lisibilité + pour l'extension des conditions